src/ArrayExpressionBuilder.php

```diff
@@ -71,19 +79,25 @@ protected function buildPlaceholders(ExpressionInterface $expression, array &$pa
         $value = $expression->getValue();
 
         $placeholders = [];
-        if ($value === null || (!is_array($value) && !$value instanceof Traversable)) {
+        if (!is_array($value) && !$value instanceof Traversable) {
```

Comment:
I think this test covers it, anyway if you have a way to make it fail please add the tests.

https://github.com/terabytesoftw/db-pgsql/blob/2222940aa94f1364177f3df8639707902d2c3d7f/tests/ColumnSchemaTest.php#L34

Reply:
Should be similar to other methods but passing `new ArrayExpression(null)` there. Actually, it could be fixed by using types:

```php
class ArrayExpression implements ExpressionInterface, \ArrayAccess, \Countable, \IteratorAggregate
{
    private ?string $type = null;
    private array $value;
    private int $dimension;

    public function __construct(array $value = [], $type = null, $dimension = 1)
```

Reply:
Fixed, add check for null.